backend/src/middleware/error.ts

```diff
@@ -1,11 +1,13 @@
+import { internalServerError } from '../errors'
+
 /**
  * A Pages middleware function that logs errors to the console and responds with 500 errors and stack-traces.
  */
 export async function errorHandling(context: EventContext<unknown, any, any>) {
 	try {
 		return await context.next()
 	} catch (err: any) {
-		console.log(err.stack)
-		return new Response(`${err.message}\n${err.stack}`, { status: 500 })
+		const error = err instanceof Error ? err : new Error('Internal Server Error')
```

Comment:
I don't think throwing an non Error is going to happen, it would be a bug.

Reply:
yeah I agree, I was being a bit defensive here, I don't mind just casting to `Error` 🙂 👍